python/private/builders_util.bzl

```diff
@@ -15,6 +15,41 @@
 """Utilities for builders."""
 
 load("@bazel_skylib//lib:types.bzl", "types")
+load(":bzlmod_enabled.bzl", "BZLMOD_ENABLED")
+
+def normalize_transition_inout_values(arg_name, values):
+    """Normalize transition inputs/outputs to canonical label strings."""
+    for i, value in enumerate(values):
+        values[i] = normalize_transition_inout_value(arg_name, value)
+
+def normalize_transition_inout_value(arg_name, value):
```

Comment:
`inout` reads very similar to `input`. I read it that way multiple times in my head. Maybe we should rename it or make it `in_out` instead? `io` would also work but has other connotations.

Reply:
Done.